docs/how_to_guides/authentication_using_credential_helper.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Authentication with OCI Registries
```

Comment:
The title is about OCI Registry, but the body is about docker specific. Can it be more generic, or it is just the current state?

Reply:
ORAS can authenticate with the OCI registries by reading the credentials stored in the Credential helper. Docker credential helper is one of the credential stores.